zerver/lib/upload.py

```diff
@@ -165,6 +168,23 @@ def upload_image_to_s3(
 
     key.set_contents_from_string(force_str(contents), headers=headers)
 
+def get_total_qouta(user):
+    # type: (UserProfile) -> int
+    uploads = Attachment.objects.filter(owner=user)
+    total_quota = 0
+
+    for item in uploads:
+        total_quota += item.size
+    return total_quota
```

Comment:
You might be able to do a nicer query with the Django ORM here to do this in one step (https://docs.djangoproject.com/en/1.10/topics/db/aggregation/#cheat-sheet)